Content.Server/Radio/Components/HeadsetComponent.cs

```diff
@@ -1,19 +1,38 @@
 using Content.Server.Radio.EntitySystems;
 using Content.Shared.Inventory;
-using Content.Shared.Radio;
-using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.Set;
+using Robust.Shared.Audio;
+using Robust.Shared.Containers;
 
 namespace Content.Server.Radio.Components;
-
 /// <summary>
 ///     This component relays radio messages to the parent entity's chat when equipped.
 /// </summary>
 [RegisterComponent]
 [Access(typeof(HeadsetSystem))]
 public sealed class HeadsetComponent : Component
 {
-    [DataField("channels", customTypeSerializer: typeof(PrototypeIdHashSetSerializer<RadioChannelPrototype>))]
-    public readonly HashSet<string> Channels = new() { "Common" };
+    [ViewVariables]
+    public HashSet<string> Channels = new();
+
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("keySlotsAmount")]
+    public int KeySlotsAmount = 2;
+ 
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("isKeysExtractable")]
+    public bool IsKeysExtractable = true;
```

Comment:
Neither of these variables are ever used on the headset prototypes, which is the only place they'll actually work as intended.